tests/test_eccv_conference.py

```diff
@@ -1016,7 +1016,7 @@ def test_desk_reject_submission(self, conference, client, test_client, selenium,
 
     def test_withdraw_submission(self, conference, client, test_client, selenium, request_page, helpers):
 
-        conference.setup_post_submission_stage(force=True)
+        conference.setup_post_submission_stage(force=True, hide_fields=['_bibtex'])
```

Comment:
I believe the submission doesn't have a _bibtex value, can you try to hide another field? 

Didn't NeurIPS test work?

Reply:
Withdraw submission does have the `_bibtex` field, however, desk-reject doesn't.

No, NeurIPS didn't work. It had a completely different error.

Reply:
I added back the NeurIPS test and fixed the error. The Withdrawn and Desk Rejected invitations must have the hidden fields declared in their reply.